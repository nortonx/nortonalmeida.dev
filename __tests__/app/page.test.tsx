import { render, screen } from "@testing-library/react";
import Page from "@/app/page";
import { describe, it, expect } from "@jest/globals";

describe("Home Page", () => {
  it("should render the page and match snapshot", async () => {
    const { container } = render(<Page />);
    const page = await screen.findByTestId("home-page");
    expect(page).toBeDefined();
    expect(container).toMatchSnapshot();
  })
})
