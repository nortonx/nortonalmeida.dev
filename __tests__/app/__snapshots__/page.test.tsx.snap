// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render the page and match snapshot 1`] = `
<div>
  <div
    data-testid="home-page"
  >
    Shadcn lives
  </div>
</div>
`;
